portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -72,4 +80,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     
     response.sendRedirect("comments.html");
   }
+
+  /** Returns the max number of comments entered by the user */
+  private int getMaxComments(HttpServletRequest request) {
+    // Get the input from the form.
+    String maxNoCommentsString = request.getParameter("maxNoComments");
+
+    // Convert the input to an int.
```

Comment:
This comment is redundant. `Integer.parseInt(maxNoCommentsString);` is pretty self-explanatory.
```suggestion
```